pennylane/tape/qscript.py

```diff
@@ -438,6 +449,25 @@ def _update_observables(self):
                     self._obs_sharing_wires.append(m.obs)
                     self._obs_sharing_wires_id.append(i)
 
+    def _update_measured_wires(self):
+        """Updates the measured wires straightforwardly.
+
+        Sets:
+            _measured_wires (int): The wires measured in this QuantumScript
+        """
+
+        def extract_wires(obs):
+            for wire in obs.wires:
+                if wire not in self._measured_wires:
+                    self._measured_wires.append(wire)
```

Comment:
Might be good to point you toward [`Wires.all_wires`](https://docs.pennylane.ai/en/stable/code/api/pennylane.wires.Wires.html#pennylane.wires.Wires.all_wires)

Reply:
Thank you, I can indeed use `Wires.all_wires` here instead if you prefer.